Source/WebKit/UIProcess/ios/WKContentViewInteraction.mm

```diff
@@ -4899,27 +4898,20 @@ - (void)doAfterComputingImageAnalysisResultsForBackgroundRemoval:(CompletionHand
     });
 }
 
-- (BOOL)_handleTapOverImageAnalysisInteractionButton:(CGPoint)position
+- (BOOL)_handleTapOverInteractiveControl:(CGPoint)position
 {
-    if (!_imageAnalysisInteraction)
+    auto *hitButton = dynamic_objc_cast<UIControl>([self hitTest:position withEvent:nil]);
+    if (!hitButton)
         return NO;
 
-    auto *hitButton = dynamic_objc_cast<UIButton>([self hitTest:position withEvent:nil]);
+    // If the control is the image analysis button, it has to do this as well
     UIButton *analysisButton = [_imageAnalysisInteraction analysisButton];
-    // FIXME: Instead of a class check, this should be a straightforward equality check.
-    // However, rdar://91828384 currently prevents us from doing so. We can simplify this logic
-    // once the fix for that bug has landed.
```

Comment:
Why are we removing this comment?

Reply:
I originally removed it because the associated radar is closed. I have since updated it.